convert2rhel/cert.py

```diff
@@ -73,4 +74,8 @@ def remove(self):
             os.remove(self._target_cert_path)
             loggerinst.info("Certificate %s removed" % self._target_cert_path)
         except OSError as err:
-            loggerinst.error("OSError({0}): {1}".format(err.errno, err.strerror))
+            if err.errno == errno.ENOENT:
+                # Resolves RHSM error when removing certs, as the system might not have installed any certs yet
+                loggerinst.debug("No RHSM certificates found to be removed")
```

Comment:
Oh, I see why. It should be `loggerinst.info`, not `loggerinst.debug` so that it's printed even without using the `--debug` option. It's like that for all other TASK-type messages for the reason that we want to have at least some kind of status message printed under each TASK before getting to another TASK.

Reply:
I did the change under https://github.com/oamg/convert2rhel/pull/447/commits/adaf87ba45573fc33b8f4416bee2a871e9f15a90.